app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java

```diff
@@ -156,11 +157,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     private void sortAttendees(int sortBy) {
         if (sortBy == SORTBYTICKET) {
-            fastItemAdapter.withComparator((Attendee a1, Attendee a2) -> a1.getTicket().getType().compareTo(a2.getTicket().getType()));
+            ((ComparableItemListImpl<Attendee>) fastItemAdapter.getItemList()).withComparator((Attendee a1, Attendee a2) -> a1.getTicket().getType().compareTo(a2.getTicket().getType()), true);
```

Comment:
Everything except this looks good. Find a way you don't need to cast

Reply:
Couldn't find any other way. Check this: https://github.com/mikepenz/FastAdapter/issues/675#issuecomment-392546388